### PR TITLE
 Update GDS User UI Translations

### PR DESCRIPTION
**Pull request by rebeccabilbro**
### Scope of changes

This PR adds the translations of the PO files for the GDS User UI. 

- [x] french
- [x] german
- [x] chinese
- [x] japanese

### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

```
Catalog statistics for src/locales/{locale}/messages:
┌─────────────┬─────────────┬─────────┐
│ Language    │ Total count │ Missing │
├─────────────┼─────────────┼─────────┤
│ de          │     533     │   208   │
│ en (source) │     501     │    -    │
│ fr          │     503     │   155   │
│ ja          │     535     │   212   │
│ zh          │     533     │   208   │
│ en-dh       │     520     │   40    │
└─────────────┴─────────────┴─────────┘
```

### Author checklist

- [ ] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [x]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [x] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [x] Check the translation statistics

### CODE REVIEW

File: web/gds-user-ui/src/locales/de/messages.po [hunk 1]
```diff
@@ -203,7 +203,7 @@ msgid "Already have an account?"
 msgstr ""
 
 #: src/components/NetworkAnnouncements/index.tsx:21
-msgid "An error has occurred to load annoucements"
+msgid "An error has occurred to load announcements"
```
Comment (rebeccabilbro) [inline web/gds-user-ui/src/locales/de/messages.po:206]: @elysee15 @masskoder  this appears to be a typo in the app - can you please fix?

File: web/gds-user-ui/src/locales/fr/messages.po [hunk 2]
```diff
@@ -1743,60 +1690,56 @@ msgstr ""
...
-msgstr ""
+msgstr "Point de terminaison TRISA: MainNet"
 
 #: src/components/TrisaImplementation/index.tsx:72
 msgid "TRISA Endpoint: TestNet"
-msgstr ""
+msgstr "Point de terminaison TRISA: TestNet"
 
 #: src/components/OpenSourceResources/index.tsx:15
-msgid "TRISA Githubs repo"
```
Comment (rebeccabilbro) [inline web/gds-user-ui/src/locales/fr/messages.po:1773]: @elysee15 @masskoder another typo - can you please fix in the app code?

It should be `TRISA Github repo`

Comment (bbengfort) [reply web/gds-user-ui/src/locales/fr/messages.po:1773]: Fixed in this PR.

**Review by bbengfort — COMMENTED** [review]
I've fixed the typos, added the dropdown back in, and merged in the translated PO files running `lingui extract` and `lingui compile` to merge the changes with the latest code base. The translation statistics are updated in the description of the story. I've performed very basic QA with `git difftool` and did not QA any of the language elements themselves. 

I propose that we deploy this to staging for QA.

**Review by bbengfort — APPROVED** [review]

**Comment by bbengfort** [comment]
@masskoder could you help with getting the tests to pass?

**Comment by masskoder** [comment]
Sure I'm working on it

**Review by masskoder — APPROVED** [review]
@bbengfort  Looks good.

**Pull request merged**